packages/vue3/src/_dev/App.vue

```diff
@@ -1,134 +1,39 @@
 <script setup lang="ts">
 import 'uno.css';
-import '~/assets/main.css';
+import '../assets/main.css';
```

Comment:
Why did we remove the relative path here?
```suggestion
import '~/assets/main.css';
```